docs/config.rst

```diff
@@ -13,46 +13,89 @@ Defining sessions
 .. autofunction:: nox.session
 
 Nox sessions are configured via standard Python functions that are decorated
-with ``@nox.session`` or start with ``session_``. For example, these are all
-sessions::
+with ``@nox.session``. For example::
 
     import nox
 
-    def session_a(session):
+    @nox.session
+    def b(session):
```

Comment:
Nitpick: The name `b` no longer makes sense as there is now only one sample. Perhaps `unit_tests` or something.